docs/modules/servers/pages/distributed/configure/opensearch.adoc

```diff
@@ -97,6 +97,11 @@ turning off headers indexing result in non-strict compliance with the IMAP / JMA
 If the message is not found, it will fall back to the old behavior (The message will be indexed from the blobStore source)
 Default to false.
 
+| opensearch.text.fuzziness.search
+| Use fuzziness on text searches. This option helps to correct user typing mistakes.
+
+Default to true.
```

Comment:
I'm peacemealed: have this been performance tested both on the indexing and search part?

Until it is I would rather be defensive and keep this disabled by default.

Reply:
> on the indexing and search part?

This change only affects the search phase.

> have this been performance tested

Benchmark on Email/query with random text: https://github.com/linagora/james-gatling/blob/master/src/main/scala-2.13/org/apache/james/gatling/jmap/rfc8621/JmapEmail.scala#L49

Fuzzy search off:
![image](https://github.com/user-attachments/assets/f9ca47db-55c7-4202-94be-0a7f1c525734)

Fuzzy search on:
![image](https://github.com/user-attachments/assets/5c30eac3-e51e-47ad-8196-783de989cf22)

p99 94ms -> p99 173ms: indeed with fuzzy on, the OpenSearch query is slower, but it is still acceptable IMO.

Reply:
The email query shall exercise text search for this to be relevant. Is it the case?

Reply:
Yes it is.